### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for button in "Container Registry-Open ACR Explorer dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1411]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Push Image.
4. In "push Image" dialog ,fill all the required information and select ok button.
5. Navigate to and select "Show Azure Explorer" button at the toolbar .
6. In "Azure Explorer" , select the pushed image under container registries and use shift+F10 to open the context menu.
7. Navigate to and select "Open ACR Explorer" menu item .
8. Tab to refresh button and observe the name property using inspect.
9. Tab to previous page and next page buttons and observe the name property using inspect

**Actual Result:**
Name property is empty for the button 

**Narrator behaviour:**
Tab
button

**NVDA behaviour:**
toolbar
button

**Expected Result:**
Name property should be simple and meaningful.
Name property for the button should be "Refresh".
Name property should be Previous page and Next page

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![acr explorer-buttons-name-empty](https://user-images.githubusercontent.com/22956130/35685961-249e7536-0791-11e8-98f0-5b9baabce7c4.jpg)
![acr explorer-buttons-name-empty-nvda](https://user-images.githubusercontent.com/22956130/35685965-275f994e-0791-11e8-8644-d2b5e271f6cd.jpg)

**Comment by jdneo** [comment]
Fixed

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1411-fixed](https://user-images.githubusercontent.com/22956130/39760618-65769bb2-52f3-11e8-8b24-de98e8ca7bd6.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749876
